fcrepo-jms/src/test/java/org/fcrepo/integration/jms/observer/AbstractJmsIT.java

```diff
@@ -119,61 +125,68 @@ public void testIngestion() throws RepositoryException {
 
         LOGGER.debug("Expecting a {} event", RESOURCE_CREATION.getType());
 
-        final Session session = repository.login();
+        final FedoraSession session = repository.login();
+        session.addSessionData(BASE_URL, TEST_BASE_URL);
```

Comment:
This has to do with the following: at present, we are passing data from the HTTP layer into the event system, particularly `User-Agent` and `Host` header values. Those values are passed into the event system via the current `jcr.Session`. In this PR, we replace the `jcr.Session` with a `FedoraSession`, but the mechanism is the same -- this user data is passed into the FedoraSession in the HTTP layers and then retrieved by the eventing machinery.

In these integration tests, however, there is no HTTP layer, so I'm adding that data manually. I _could_ omit that data and a warning would be issued (given that no `baseUrl` or `userAgent` values are available), but I wanted to avoid those warnings.